pygame_gui/elements/ui_form.py

```diff
@@ -0,0 +1,1144 @@
+    :param starting_height: The starting layer height of this container above its container.
+                            Defaults to 1.
+    :param container: The container this container is within. Defaults to None (which is the root
+                      container for the UI)
+    :param parent_element: A parent element for this container. Defaults to None, or the
+                           container if you've set that.
+    :param object_id: An object ID for this element.
+    :param anchors: Layout anchors in a dictionary.
+    :param visible: Whether the element is visible by default. Warning - container visibility
+                    may override this.
+    """
+
+    def __init__(self,
+                 relative_rect: pygame.Rect,
+                 form: "UIForm",
+                 name: str,
+                 questionnaire: Mapping[str, Union[str, Mapping]],
+                 manager: Optional[IUIManagerInterface] = None,
+                 *,
+                 starting_height: int = 1,
+                 container: Optional[IContainerLikeInterface] = None,
+                 parent_element: Optional[UIElement] = None,
+                 object_id: Optional[Union[ObjectID, str]] = None,
+                 anchors: Optional[Dict[str, Union[str, UIElement]]] = None,
+                 visible: int = 1):
+        super().__init__(relative_rect,
+                         resize_left=False,
+                         resize_right=False,
+                         resize_top=False,
+                         manager=manager,
+                         starting_height=starting_height,
+                         container=container,
+                         parent_element=parent_element,
+                         object_id=object_id,
+                         anchors=anchors,
+                         visible=visible)
+
+        self._create_valid_ids(container=container,
+                               parent_element=parent_element,
+                               object_id=object_id,
+                               element_id="section")
+
+        self.form = form
+        self.name = name
+
+        self.background_colour = None
+        self.border_colour = None
+        self.background_image = None
+        self.border_width = 1
+        self.shadow_width = 2
+        self.padding = (5, 5)
+
+        self.label_height = 30
+        self.section_label_height = 30
+        self.field_height = 40
+        self.large_field_height = 80
+        self.gap = 10
+        self.label_gap = 0
+        self.section_gap = 15
+        self.section_indent = 5
+
+        self.shape = "rectangle"
+        self.shape_corner_radius = 0
+
+        anchors = {"left": "left", "right": "right", "top": "top"}
+
+        # You can click this button to expand/hide the container
+        self.main_expand_button = UIButton(pygame.Rect(0, 0, 0, 0),
+                                           text=self.name + " ▼",
```

Comment:
```suggestion
                                           text=self.name + " ▼",
```

Slightly annoyingly the new default font, while otherwise better than the old default font, doesn't support the same range of symbols. I kind of want to add HTML parsing to button text just to let us fix this use case by switching to the old default font midway through this text.

Anyway, right now it won't work unless we style this whole button using 'fira_code_symbols'.